### ISSUE TITLE
Stop Midi-Map from automatically advancing to next mapping

### ISSUE THREAD

**Issue by bgracie** [VCVRack/Rack#1696]
### Summary

Currently, Midi-Map advances to the next mapping slot after the parameter and control CC are registered.  This makes it impossible to adjust the parameter before automatically advancing to the next mapping.  This kind of adjustment is often necessary, since the external control is likely at a different position than the internal parameter was before the mapping.

### Proposed design

Don't automatically advance to the next mapping.

OR

Add a short (~1-2s) timeout after the last external CC message is received before advancing to the next mapping.

**Comment by AndrewBelt** [comment]
Yeah, this is a bug because it shouldn't activate if the same CC number is detected after setting a mapping. Fixing this bug *wouldn't* disable adding multiple mappings with the same CC, it would just require explicitly clicking on the mapping, which is acceptable.
